MultiplayerTestRunner/src/Main.cpp

```diff
@@ -0,0 +1,115 @@
+#include "../include/ErrorCodes.h"
+#include "../include/ProcessDescriptors.h"
+#include "CLIArgs.h"
+#include "LoginRAII.h"
+#include "RunnableTests/CreateAvatar.h"
+#include "SpaceRAII.h"
+#include "Utils.h"
+
+#include <CSP/Common/String.h>
+#include <chrono>
+#include <string>
+#include <thread>
+
+#ifdef RUN_MULTIPLAYER_RUNNER_TESTS
+	#include <gtest/gtest.h>
+#endif
+
+/********************************
+ * MULTIPLAYER TEST RUNNER
+ * This is a CLI application designed to launch specified tests on independent processes (as CSP currently supports only one user per process), in
+ * order to support testing multiple clients on multiple processes interacting in the same room.
```

Comment:
[nit] "... in the same space"

Reply:
https://github.com/magnopus-opensource/connected-spaces-platform/pull/558/commits/f731d167c1b5afe99b618af098c2c83797ee78d0